### DISCUSSION TITLE
Hi, developer~ How to convert the screen coordinate to Model's position?

### DISCUSSION THREAD

**Discussion by amin2312** [jnsmalm/pixi3d#18] [Q&A] [answered]
I can't find the related API.
I need your help, thank you very mush!

**Comment by jnsmalm** [comment, marked as answer]
Hey,

you can check here:

https://api.pixi3d.org/classes/camera.html#screentoworld
https://api.pixi3d.org/classes/camera.html#worldtoscreen

If you are using the main camera you can do this:

`let screenCoords = Camera.main.worldToScreen(0, 0, 0)`

**Reply by amin2312** [reply]
thank you very much
